app/src/main/res/values/strings.xml

```diff
@@ -429,16 +429,19 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="link_indoorequal_description">Discover indoor maps of malls, railway stations etc.</string>
     <string name="link_wheelmap_description">A map for finding wheelchair-accessible places</string>
     <string name="link_osm_buildings_description">A map that shows buildings in 3D</string>
-    <string name="link_openvegemap_description">Discover vegetarian and vegan restaurants in your city</string>
     <string name="link_touch_mapper_description">Create tactile maps easily for any address</string>
     <string name="link_mapy_tactile_description">Tactile maps for the visually impaired in various scales</string>
     <string name="link_josm_description">Fully-featured OSM editor for the desktop. This is the power tool for regular contributors.</string>
     <string name="link_vespucci_description">Advanced OSM editor for Android</string>
     <string name="link_ideditor_description">Simple OSM editor for the browser. Optimized for the desktop or large tablets</string>
+    <string name="link_every_door_description">Powerful POI editor for Android and iOS</string>
+    <string name="link_mapcomplete_description">Collection of thematic OSM maps that each show and let you edit one kind of feature</string>
     <string name="link_umap_description">Create maps with custom data quickly and embed them in your site</string>
     <string name="link_opnvkarte_description">A map of public transport routes and stops</string>
     <string name="link_pic4review_description">Contribute to OSM simply by looking at pictures of your city</string>
     <string name="link_wiki_description">The wiki is your starting point to everything related to OpenStreetMap</string>
+    <string name="link_forum_description">Need help? Want to read what’s going on? Come and mingle, it’s bustling with activity from communities all over the globe!</string>
```

Comment:
Would it make sense to mention that it is (1) an official forum (2) you can login with OSM account?

Reply:
Well, it's called "OpenStreetMap Community Forum", so the expectation for both is "yes". The wiki link also does not have any description that it is official.